netutils/webclient/webclient.c

```diff
@@ -132,6 +129,37 @@
  * Private Types
  ****************************************************************************/
 
+enum webclient_state
```

Comment:
```suggestion
enum webclient_state_e
```
According to the coding style, enumeration names must end with the suffix `_e`.

Reply:
fixed